Bottom: e80dbd662d1ac9e822727e9abf283c61aa771328
Top:    38dcb64a316019d564ed9277cc74dc6c59d92b43
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-16 13:34:48 -0400

Refresh of efika-stac9766

---

diff --git a/sound/soc/fsl/efika-stac9766.c b/sound/soc/fsl/efika-stac9766.c
index 39b3693..6bfa498 100644
--- a/sound/soc/fsl/efika-stac9766.c
+++ b/sound/soc/fsl/efika-stac9766.c
@@ -216,7 +216,7 @@ static struct snd_soc_dai_link efika_dai[] = {
 
 static struct snd_soc_card efika = {
 	.name = "Efika-STAC9766",
-	.platform = &mpc5200_soc_platform,
+	.platform = &fsl_soc_platform,
 	.dai_link = efika_dai,
 	.num_links = ARRAY_SIZE(efika_dai),
 };
